lib/langchain/vectorsearch/epsilla.rb

```diff
@@ -127,7 +127,7 @@ def similarity_search_by_vector(embedding:, k: 4)
     # @param k [Integer] The number of results to have in context
     # @yield [String] Stream responses back one String at a time
     # @return [String] The answer to the question
-    def ask(question:, k: 4, &block)
+    def ask(question:, k: 4, &)
```

Comment:
I don't exactly understand why but under different versions of Ruby this keeps throwing an error when it's an "anonymous block"

Reply:
yeah. it's the auto linter. I'll revert it